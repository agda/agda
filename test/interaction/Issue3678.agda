open import Agda.Builtin.Sigma
open import Agda.Builtin.Equality

postulate
  A : Set
  B : A → Set
  a : A

Pi : (A → Set) → Set
Pi B = {x : A} → B x

foo : Pi \ y →  Σ (B y) \ _ → Pi \ z → Σ (y ≡ a → B z) \ _ → B y → B z → A
foo = {!!} , (\ { refl → {!!} }) , {!!}
