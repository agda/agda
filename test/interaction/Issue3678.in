top_command (cmd_load currentFile [])
goal_command 2 (cmd_context AsIs) ""
