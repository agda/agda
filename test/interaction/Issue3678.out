(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : B x ?1 : B x₁ ?2 : B x → B x₁ → A " nil)
((last . 1) . (agda2-goals-action '(0 1 2)))
(agda2-status-action "")
(agda2-info-action "*Context*" "x : A (not in scope) x₁ : A (not in scope)" nil)
